README.md

```diff
@@ -20,16 +29,25 @@ is used and required for dependencies
 **Building:**
 
 ```sh
-$ go build ./cmd/...
+$ ./hack/build.sh
```

Comment:
what's the advantage of running the build in /hack versus just go build?

Reply:
I think we might add additional commands to build in the future; for example, we could force re-generation.